cairo/src/helper.rs

```diff
@@ -0,0 +1,99 @@
+//! This module inlcudes some field helpers that are useful for Cairo
+
+use ark_ff::FftField;
+use o1_utils::FieldHelpers;
+
+//(TODO move to utils inside FieldHelpers)
+
+/// Field element helpers for Cairo
+pub trait CairoFieldHelpers<F: FftField> {
+    /// Return field element as byte, if it fits. Otherwise returns least significant byte
+    fn ls_byte(self) -> u8;
```

Comment:
better to spell this out `least_significant_byte` or use common acronym `lsb`

Reply:
Ah, well I thought `lsb` would stand for least significant bit instead of byte. In any case, I can switch to the verbose format if that makes more sense.

Reply:
Maybe lsb only stands for bit, not sure :D I feel like I've seen it being used for both but I might be wrong. In any case the return value (u8 instead of bool) and documentation would be enough to indicate that it's a byte

Reply:
Oh good point... I honestly prefer `lsb` so I will be changing it in a future PR